### PR TITLE
support multi contianer_name

### PR DESCRIPTION
**Pull request by hussein-firas**
replace container_name with container_names to support multiple container_name in each file

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -31,7 +31,7 @@ runs:
   image: 'docker://ghcr.io/loveholidays/gitops-action-yaml-updater:v1.7.0'
   args:
     - ${{ inputs.mode }}
-    - ${{ inputs.container-name }}
+    - ${{ inputs.container-names }}
```
Comment (dwilliams782) [inline action.yml]: This will need to stay as container-name to avoid having to change it everywhere

File: action.yml [hunk 2]
```diff
@@ -5,8 +5,8 @@ inputs:
     description: 'TYPE of the new value to be updated'
     required: true
     default: 'IMAGE_TAG'
-  container-name:
-    description: 'Name of the container'
+  container-names:
```
Comment (dwilliams782) [inline action.yml]: This will need to stay as container-name to avoid having to change it everywhere

File: action.yml (hunk 1 above)
Comment (hussein-firas) [reply action.yml]: @dwilliams782  
and keep the value comma separated names? just keeping the variable name?

Comment (dwilliams782) [reply action.yml]: Yeah - you can go with the plural name but you'll need to update it anywhere it's used

**Pull request closed without merging**